systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderIsolatedST.java

```diff
@@ -59,6 +59,14 @@
 import static io.strimzi.systemtest.Constants.ACCEPTANCE;
 import static io.strimzi.systemtest.Constants.CONNECT;
 import static io.strimzi.systemtest.Constants.CONNECT_COMPONENTS;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_CLASS_NAME;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_CONNECTOR_NAME;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_FILE_NAME;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_JAR_CHECKSUM;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_JAR_URL;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_JAR_WRONG_CHECKSUM;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_TGZ_CHECKSUM;
+import static io.strimzi.systemtest.Constants.ECHO_SINK_TGZ_URL;
```

Comment:
Maybe you can use the `Constants.X`, so you will not have to import all of those. But it's just a nit